runtime/doc/eval.txt

```diff
@@ -3771,6 +3772,10 @@ hasmapto({what} [, {mode} [, {abbr}]])			*hasmapto()*
 <		This installs the mapping to "\ABCdoit" only if there isn't
 		already a mapping to "\ABCdoit".
 
+hastabdir()						*hastabdir()*
+		The result is a Number, which is 1 when the current
+		tab has set a local path via |:tcd|, and 0 otherwise.
+
```

Comment:
I like the `0,1,2,3` idea. 

- It is backwards compatible
- It avoids a new function
- It emphasizes that tab-local is a special case of local.